Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Microsoft Translator Service/MicrosoftTranslatorPage.xaml.cs

```diff
@@ -0,0 +1,111 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Microsoft.Toolkit.Uwp.Services.MicrosoftTranslator;
+using Windows.System;
+using Windows.UI.Xaml;
+
+namespace Microsoft.Toolkit.Uwp.SampleApp.SamplePages
+{
+    /// <summary>
+    /// A sample XAML page that shows how to use <see cref="TranslatorServiceClient"/> class.
+    /// </summary>
+    public sealed partial class MicrosoftTranslatorPage
+    {
+        private TranslatorServiceClient translatorClient;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MicrosoftTranslatorPage"/> class.
+        /// </summary>
+        public MicrosoftTranslatorPage()
+        {
+            InitializeComponent();
+
+            translatorClient = new TranslatorServiceClient();
+        }
+
+        private async void GetKey_OnClick(object sender, RoutedEventArgs e)
+        {
+            await Launcher.LaunchUriAsync(new Uri("https://portal.azure.com/#create/Microsoft.CognitiveServices/apitype/TextTranslation"));
+        }
+
+        private async void GetLanguages_OnClick(object sender, RoutedEventArgs e)
+        {
+            if (!await Tools.CheckInternetConnectionAsync())
```

Comment:
You could have used the ConnectionHelper class that the community implemented instead of adding a new one :) could you please used the ConnectionHelper for consistency